content/syllabuses/data-sci.md

```diff
@@ -9,29 +9,29 @@ _db_id: 2
 - {{% contentlink path="topics/solo-learn/python/module-5-more-types" %}}
 - {{% contentlink path="projects/katas/level-2" flavour="python" %}}
 - {{% contentlink path="language-agnostic/intro-to-hackerrank" flavour="python" %}}
-- {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
 - {{% contentlink path="workshops/general-coding-best-practices" %}}
-- {{% contentlink path="projects/data-science-specific/statistical-thinking" flavour="python" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/automated-testing-with-pytest"%}}
-- {{% contentlink path="projects/data-science-specific/data-visualisation" %}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
+- {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
+- {{% contentlink path="projects/data-science-specific/statistical-thinking" flavour="python" %}}
+- {{% contentlink path="projects/data-science-specific/data-visualisation" %}}
 - {{% contentlink path="topics/survey-design" %}}
 - {{% contentlink path="topics/google-forms-like-a-boss" %}}
 - {{% contentlink path="projects/data-science-specific/cross-validation-and-simple-linear-regression/" flavour="python" %}}
 - {{% contentlink path="projects/morse-code" flavour="python" %}}
 - {{% contentlink path="workshops/intro-to-data-ethics" %}}
+- {{% contentlink path="topics/data-ethics-and-privacy" %}}
 - {{% contentlink path="projects/tdd/string-calculator" flavour="python" %}}
```

Comment:
String calc is good because it makes people very comfortable with regex and tdd. But maybe it's a bit heavy heavy handed. I think regex is important for a data-scientist - I have to use regex a lot when wrangling stuff. But the regex I use tends to be very simple. 

Would a simpler project do? Or can we move this project to later? I'm tinking of getting a simpler project for the other streams as well.

Reply:
I agree it's hard but I think string-calc is very valuable, there's lots learning that happens working through it. I'd be in favour of maybe simplifying it a bit, but I'm hesitant about removing it.